lib/coverband/utils/tasks.rb

```diff
@@ -1,17 +1,24 @@
 # frozen_string_literal: true
 
 namespace :coverband do
-  Coverband.configure
+  # TODO: FixMe before releasing 4.2.1
+  # This was previously needed when Coverband was generally require false
+  # now this would double configure, which isn't a good thing
+  # I think we need ot make it safe to call configure twice and no-op if it has been called
+  # because removing this means if you have require: false on coverband or
+  # COVERBAND_DISABLE_AUTO_START=true the rake tasks wouldn't work
+  # Coverband.configure
 
   def environment
+    Coverband.configuration.report_on_exit = false
```

Comment:
Think we should disable background thread also.

Reply:
good call done